### DISCUSSION TITLE
Does BreakoutProcess support paths like BreakoutFolder?

### DISCUSSION THREAD

**Discussion by lwcorp** [sandboxie-plus/Sandboxie#1905] [Various ideas and suggestions]
https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.8:
> added "BreakoutProcess=program.exe", with this option selected applications can be started unboxed from within a box #1500
> -- Note: "BreakoutFolder=some\path" is also available

Does `BreakoutProcess` support paths just like `BreakoutFolder`? Can it be used like `BreakoutProcess=some\path\program.exe`?

**Comment by DavidXanatos** [comment]
no currently it only supports program.exe without a path

**Reply by lwcorp** [reply]
Thanks, so create  #1907.
